ui/src/app/cron-workflows/components/cron-workflow-details/cron-workflow-details.scss

```diff
@@ -0,0 +1,13 @@
+@import 'node_modules/argo-ui/src/styles/config';
+
+.workflows-list {
+  &__status {
+    max-width: 80px;
+    display: flex;
+    align-items: center;
+
+    &--checkbox {
+      appearance: none;

```

Comment:
interesting workaround to just hide the checkbox in CSS!

2 things though:
1. Can we add a comment that the checkbox is not used on this page?
1. The CSS is currently global (unfortunately), so I think this could break other pages with `.workflows-list`. To fix that, we can just ensure that the parent is the Cron Workflows page by adding another class on the page and selecting on it

Also this doesn't work on [older browsers](https://developer.mozilla.org/en-US/docs/Web/CSS/appearance#browser_compatibility), but those are all EoL, so I think that's fine.

Reply:
I solved the issue as below

```tsx
public render() {
        const wf = this.props.workflow;
        return (
            <div className='workflows-list__row-container workflows-cron-list__row-container'>
                <div className='row argo-table-list__row'>
                    <div className='columns small-1 workflows-list__status'>
                        <input
                            type='checkbox'
                            className='workflows-list__status--checkbox'
                            checked={this.props.checked}
                            onClick={e => {
                                e.stopPropagation();
                            }}
                            onChange={e => {
                                this.props.select(this.props.workflow);
                            }}
                        />
```
I add className `workflows-cron-list__row-container'` in `WorkflowsRow` component to hide checkbox!

```css
.workflows-cron-list {
    &__status {
        max-width: 80px;
        display: flex;
        align-items: center;

        /* checkboxes are not visible and are unused on this page */
        &--checkbox {
            appearance: none;
        }
    }
}
```
Also, I select`.workflows-cron-list` in CSS

Reply:
This is roughly the right approach, but you need a class that is located in the `CronWorkflowDetails` component, as it has to be _unique_ to that list and not used in any other component's lists. See my in-line comment as well

Reply:
I was thinking of using a [CSS child selector](https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator). So could still use `.workflows-list` but only those that are children of a class in `CronWorkflowDetails`. 

There are potentially other ways of doing it, but if you're not sure, I thought that suggestion might help

Reply:
I understood completely!
After your feedback, I modified like this.

```javascript
<>
                    <ErrorNotice error={error} />
                    {!workflows ? (
                        <ZeroState title='No completed cron workflows'>
                            <p> You can create new cron workflows here or using the CLI. </p>
                        </ZeroState>
                    ) : (
                        <div className='argo-table-list workflows-cron-list'>
                            <div className='row argo-table-list__head'>
                                <div className='columns small-1 workflows-list__status' />
                                <div className='row small-11'>
```
I revert `WorkflowsRow` and add className `workflows-cron-list' in the class on the same div as argo-table-list at `CronWorkflowDetails`

```css
@import 'node_modules/argo-ui/src/styles/config';

.workflows-cron-list .workflows-list {
    &__status {
        max-width: 80px;
        display: flex;
        align-items: center;

        /* checkboxes are not visible and are unused on this page */
        &--checkbox {
            appearance: none;
        }
    }
}
```
I used [descendant combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/Descendant_combinator) in CSS